radiomics/featureextractor.py

```diff
@@ -323,7 +323,11 @@ def execute(self, imageFilepath, maskFilepath, label=None):
       return featureVector
 
     # 2. Check whether loaded mask contains a valid ROI for feature extraction and get bounding box
-    boundingBox = imageoperations.checkMask(image, mask, **self.settings)
+    (boundingBox,correctedMask) = imageoperations.checkMask(image, mask, **self.settings)
```

Comment:
this should be `boundingBox, correctedMask` or `(boundingBox, correctedMask)` (mind the space), otherwise flake8 will fail (as it currently does). parenthesis are redundant.